src/main/scala/com/netflix/edda/mongo/MongoDatastore.scala

```diff
@@ -365,7 +365,8 @@ class MongoDatastore(val name: String) extends Datastore {
 
   override def remove(queryMap: Map[String, Any])(implicit req: RequestId) {
     try {
-      primary.remove(mapToMongo(queryMap))
+      var opResult = primary.remove(mapToMongo(queryMap, true))
+      logger.info(s"$req$this removed " + opResult.getN().toString + " records")
```

Comment:
Seems odd to use string interpolation for some but not all values. Should this be using placeholders instead to avoid the cost of creating the string when the log level isn't enabled?